zeroconf/test.py

```diff
@@ -686,18 +731,13 @@ def test_bad_sub_types(self):
             self.assertRaises(r.BadTypeInNameException, r.service_type_name, name)
 
     def test_good_service_names(self):
-        good_names_to_try = (
-            '_x._tcp.local.',
-            '_x._udp.local.',
-            '_12345-67890-abc._udp.local.',
-            'x._sub._http._tcp.local.',
-            'a' * 63 + '._sub._http._tcp.local.',
-            'a' * 61 + u'â._sub._http._tcp.local.',
-        )
-        for name in good_names_to_try:
-            r.service_type_name(name)
-
-        r.service_type_name('_one_two._tcp.local.', allow_underscores=True)
+        assert r.service_type_name('_x._tcp.local.') == '_x._tcp.local.'
+        assert r.service_type_name('_x._udp.local.') == '_x._udp.local.'
+        assert r.service_type_name('_12345-67890-abc._udp.local.') == '_12345-67890-abc._udp.local.'
+        assert r.service_type_name('x._sub._http._tcp.local.') == '_http._tcp.local.'
+        assert r.service_type_name('a' * 63 + '._sub._http._tcp.local.') == '_http._tcp.local.'
+        assert r.service_type_name('a' * 61 + u'â._sub._http._tcp.local.') == '_http._tcp.local.'
+        assert r.service_type_name('_one_two._tcp.local.', allow_underscores=True) == '_one_two._tcp.local.'
```

Comment:
Can you keep the previous structure, maybe convert the list from list of strings to list of 2-tuple of strings? I'm not a fan of the repeated `assert r.service_type_name(...` part. :)

Reply:
Done